kinder/hack/update-imports-order.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# GoFmt apparently is changing @ head...
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# shellcheck source=/dev/null
+source "$(dirname "$0")/utils.sh"
+# cd to the root path
+cd_root_path
+
+go install k8s.io/kubeadm/kinder/hack/orderimports
+
+
+cd "${KUBE_ROOT}"
```

Comment:
https://github.com/kubernetes/kubernetes/blob/ea0764452222146c47ec826977f49d7001b0ea8c/hack/update-bazel.sh#L20
```
KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
```
I got an error when testing this PR.
```
➜  kinder git:(order-imports) hack/update-imports-order.sh
hack/update-imports-order.sh:行31: KUBE_ROOT：未绑定的变量
```

Reply:
Thanks a lot. I forgot to delete this line of code. Fixed